### PR TITLE
Adds zipLists

### PR DESCRIPTION
**Pull request by F0rgiv**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution
    - [x] Picture of whiteboard

### CODE REVIEW

File: challenges/lib/src/test/java/challenges/linkedList/LinkedListTest.java [hunk 1]
```diff
@@ -168,4 +168,42 @@ public void LinkedListTestKthFromEndToBigException() {
         ll.append(1);
         ll.kthFromEnd(10);
     }
+
+    @Test
```
Comment (Mattpet26) [inline challenges/lib/src/test/java/challenges/linkedList/LinkedListTest.java:172]: Great work with your tests! However, don't forget to check some of the base cases. What if one linked list was empty, or both?
